src/reference/faq.rst

```diff
@@ -70,24 +68,49 @@ The other deciding factor is if you're unable to install Docker (e.g. for admins
 The ambient runtime is not available without WSL.
 
 
-.. _why-intel-miniconda-installer-on-apple-silicon:
+.. _why-conda-install-errors-on-apple-silicon:
+
+Why do I get an error when installing to my Conda environment on a Mac computer with `Apple silicon`_ (e.g. M1)?
+----------------------------------------------------------------------------------------------------------------
+
+.. _Apple silicon: https://support.apple.com/en-us/HT211814
+
+An example error:
+
+.. code-block:: text
+
+   (my-conda-env) $ mamba install augur
+
+   Looking for: ['augur']
+
+   bioconda/osx-arm64                                            No change
+
+   …
+
+   Could not solve for environment specs
+   Encountered problems while solving:
+     - nothing provides mafft needed by augur-10.0.0-py_0
+
+   The environment can't be solved, aborting the operation
+
+**TL;DR**: This happens when using an ARM64-based Conda installation, but there are workarounds.
 
-Why recommend the Intel Miniconda installer for Mac computers with Apple silicon (e.g. M1)?
--------------------------------------------------------------------------------------------
+ARM64 Apple silicon chips are great and efficient. However, many existing packages have not yet added support to run on these chips natively. An easy way to identify support on the `Bioconda packages page <https://anaconda.org/bioconda>`_ is to look for ``noarch`` or ``osx-arm64`` under the **Installers** section of a package. Without any of those, a package is not able to be installed natively on Apple silicon. This is the case for packages such as `MAFFT <https://anaconda.org/bioconda/mafft>`_ (a dependency of :term:`Augur`) and many other bioinformatics packages. For this reason, using an ARM64-based Conda installation for the average bioinformatics researcher can result in a difficult experience.
 
-.. note::
+There are two ways to work around this:
 
-   `How to tell if your Mac has an Apple silicon chip <https://support.apple.com/en-us/HT211814>`_
```

Comment:
I think we should keep this link to Apple's docs worked in somewhere contextually appropriate in the new text (not just in the heading, where it's easily missed/lost).

Reply:
Updated in [force-push](https://github.com/nextstrain/docs.nextstrain.org/compare/a651379456c5ff1e8cf35eebaa7b19fbd73c8adf..675e98db2f8e06feb21ae690edad37249fe33d89):

> This happens when using an ARM64-based Conda installation on a [computer with Apple silicon](https://support.apple.com/en-us/HT211814), but there are workarounds.

I removed the link from the heading.